### PR TITLE
Cloudflare Analytics: Fix Plans Displayed after Nudge

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

* Registers the Cloudflare Analytics feature and marks them as a feature for Premium plans (and above) 
* Ensures the nudge redirects to the Plans page with just those plans

#### Testing instructions

Presently, it's a little unclear that Cloudflare Analytics is a feature on Premium plans and higher, especially from this upsell nudge.

Screenshot 2021-03-27 at 10 22 23

In order to test this change, click that nudge (on the Stats page) and the one on `/marketing/traffic/site`.

Screenshot 2021-03-27 at 10 23 28

Now when you click those nudges, you should only see the plans which are compatible with this feature, and the Personal plan shouldn't appear (see #41044 for more details). This is consistent with what happens if you click the Google Analytics nudge on the Marketing page located right below the nudge shown above.

Screenshot 2021-03-27 at 10 23 42

cc @blackjackkent, @donlair, @sixhours

### CODE REVIEW

File: client/lib/plans/features-list.js [hunk 1]
```diff
@@ -218,6 +218,11 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'Google Analytics' ),
 	},
 
+	[ constants.FEATURE_CLOUDFLARE_ANALYTICS ]: {
+		getSlug: () => constants.FEATURE_CLOUDFLARE_ANALYTICS,
+		getTitle: () => i18n.translate( 'Cloudflare Analytics' ),
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Cloudflare Web Analytics' )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/51473)

**Comment by sixhours** [comment]
Nevermind me, wrong ticket. :P

**Comment by sixhours** [comment]
@Aurorum This works well! Can you rebase and I'll run the tests? Thank you!!

**Comment by Aurorum** [comment]
Should be done now, thank you! :)

**Review by sixhours — APPROVED** [review]

**Pull request merged**